pwa/src/pages/components/[componentId]/ComponentsDetailPage.tsx

```diff
@@ -1,8 +1,22 @@
 import { PageProps } from "gatsby";
+import _ from "lodash";
 import * as React from "react";
+import { TEMPORARY_ORGANIZATIONS } from "../../../data/organizations";
 import { ComponentsDetailTemplate } from "../../../templates/componentDetail/ComponentsDetailTemplate";
 
 const ComponentsDetailPage: React.FC<PageProps> = (props: PageProps) => {
-  return <ComponentsDetailTemplate componentId={props.params.componentId} />;
+  let TempOrganization = [];
+
+  const sample = _.sample(TEMPORARY_ORGANIZATIONS);
+
+  TempOrganization.push(sample);
+
+  return (
+    <>
+      {TempOrganization && (
+        <ComponentsDetailTemplate componentId={props.params.componentId} TempOrganization={TempOrganization} />
+      )}
+    </>
+  );
```

Comment:
In the `ComponentsDetailTemplate`, we always want to receive _one_ organisation (I think). Therefore, we never want it to be an array. I think there's some unwanted complexity here, and the following implementation also works:

```
const ComponentsDetailPage: React.FC<PageProps> = (props: PageProps) => {
  return (
    <ComponentsDetailTemplate componentId={props.params.componentId} organization={_.sample(TEMPORARY_ORGANIZATIONS)} />
  );
};
export default ComponentsDetailPage;
```

Note: I've changed the `TempOrganization` prop to `organization` because even though we're sending a temporary one, the `ComponentsDetailTemplate` doesn't have to know that.

Note: see the feedback on the `ComponentDetailTemplate`, because this new implementation will break some things there (which we'll fix in there).